packages/components/src/utils/hooks/stories/use-cx.js

```diff
@@ -28,6 +29,20 @@ const Example = ( { args, children } ) => {
 	return <span className={ classes }>{ children }</span>;
 };
 
+const ExampleWithUseMemoWrong = ( { args, children } ) => {
+	const cx = useCx();
+	// Wrong: using 'useMemo' without adding 'cx' to the dependency list.
+	const classes = useMemo( () => cx( ...args ), [ ...args ] );
```

Comment:
Assuming that we will be adding the exhaustive-deps rule soon (🤞🤞🤞), would it be worth adding a preemptive `eslint-disable-next-line` here?

Also noting that the linter will throw up this warning if we use spreads (`...args`):

> React Hook useMemo has a spread element in its dependency array. This means we can't statically verify whether you've passed the correct dependencies.

Reply:
> Assuming that we will be adding the exhaustive-deps rule soon (🤞🤞🤞), would it be worth adding a preemptive `eslint-disable-next-line` here?

Makes sense, I've added it!

> Also noting that the linter will throw up this warning if we use spreads (`...args`):

Good point. I rewrote the Storybook examples to avoid using the `args` array prop, and replace it with a `serializedStyles` prop